src/kiota/KiotaHost.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using System.Text.RegularExpressions;
+using Kiota.Builder;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace Kiota {
+    public static class KiotaHost {
+        public static RootCommand GetRootCommand()
+        {
+            var configuration = LoadDefaultConfiguration();
+            var outputOption = new Option("--output", "The ouput path of the folder the code will be generated in.") { Argument = new Argument<string>(() => "./output") };
+            outputOption.AddAlias("-o");
+            
+            var languageOption = new Option("--language", "The language to generate the code in.") { Argument = new Argument<GenerationLanguage?>(() => GenerationLanguage.CSharp) };
+            languageOption.AddAlias("-l");
+            AddEnumValidator<GenerationLanguage>(languageOption.Argument, "language");
+            var classOption = new Option("--class-name", "The class name to use the for main entry point") { Argument = new Argument<string>(() => "GraphClient") };
+            classOption.AddAlias("-c");
+            AddStringRegexValidator(classOption.Argument, @"^[a-zA-Z_][\w_-]+", "class name");
+
+            var namespaceOption = new Option("--namespace-name", "The namespace name to use the for main entry point") { Argument = new Argument<string>(() => "GraphClient") };
```

Comment:
`The namespace name to use the for main entry point` becomes
`The namespace to use for the core client class specified with the --class-name option. The default value is GraphClient.`

Same question, do we want to have a default value for this tool? I think we make this a required option without a default.

Reply:
open to a more generic name, but I'd like to leave a default here